---
title: "Research"
permalink: /research/areas/
author_profile: false
layout: research
collection: research
toc: true
toc_label: "Research Areas"
toc_icon: "flask"
toc_sticky: true
excerpt: "Translating inferences from evolutionary biology to improve human health and empower K-12 education."
header:
    overlay_image: /images/notebook.png
    overlay_filter: rgba(36, 42, 40, 0.6)
---
# About

As a lab we seek to answer the following questions:

1. [Can we predict microbial evolution during host association or towards drug resistance?](#host_drug_evo)
2. [How do bacteria evolve and form communities within biofilms? What does this teach us about nascent multicellularity?](#biofilm)
3. [How and why do ecological tradeoffs evolve?](#tradeoff)
4. [Why do genome regions replicated at different times evolve at different rates?](#genome_evo)
5. [Evolution is best taught by hands-on experimentation. How do we deliver this on a massive scale?](#evolvingstem)

<br>

<a name="host_drug_evo"></a>
# 1. Host adaptation and the evolution of antimicrobial resistance

<div class="research_fig">
    <img src="/images/squid.png" />
    <p>On the left, the dorsal view of a juvenile E. scolopes host. On the right, a schematic illustrating the stages in which host-imposed selection occurs during V. fischeri symbiosis. <br>
    <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5466423/pdf/elife-24414.pdf">Host-selected mutations converging on a global regulator drive an adaptive leap towards symbiosis in bacteria.Pankey,M.S. et al. eLife, 2017</a></p>
</div>

Our laboratory is developing the translational utility of evolutionary inference in medicine using new experimental, genomic and bioinformatic methods. Our focus is the evolution-in-action that occurs during human infections, particularly in the airway of Cystic Fibrosis patients, and we also use laboratory models of this process. We run a [sequencing facility](http://micropopbio.org/sequencing.html) with an Illumina NextSeq and Oxford Nanopore and collaborate with many investigators to identify mutations under positive selection during infections and their functional significance, with a focus on adaptations to antibiotics and to biofilm formation. We also study adaptation by symbionts to animal hosts such as the Hawaiian bobtail squid, nematodes, and the medicinal leech. This work is supported by NIH, NSF, and by Gilead. 

Interested? Check out the following publications:
    
1. [Pankey, et al. Host-selected Mutations Converging on a Global Regulator Drive on Adaptive Leap Towards Symbiosis in Bacteria. eLife, 2017](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5466423/pdf/elife-24414.pdf)
2. [Haidar, G., et al. Ceftolozane-Tazobactam for the Treatment of Multidrug-Resistant Pseudomonas aeruginosa Infections: Clinical Effectiveness and Evolution of Resistance. Clinical Infectious Diseases, 2017](https://academic.oup.com/cid/article-abstract/doi/10.1093/cid/cix182/3052769/Ceftolozane-tazobactam-for-the-treatment-of?redirectedFrom=fulltext)
3. [Honsa, E.S., et al. RelA Mutant Enterococcus faecium with Multiantibiotic Tolerance Arising in an Immunocompromised Host. mBio, 2017](http://mbio.asm.org/content/8/1/e02124-16.full)
4. [Silva, I.N., et al. Long-Term Evolution of Burkholderia multivorans during a Chronic Cystic Fibrosis Infection Reveals Shifting Forces of Selection. mSystems, 2016](http://msystems.asm.org/content/msys/1/3/e00029-16.full.pdf)

<br>

***

<br>

<a name="biofilm"></a>

# 2. Biofilms and the origins of multicellularity

<div class="research_fig">
    <img src="/images/res.png" />
    <p>Biofilm Under a Confocal Lens</p>
</div>


Check out our [MicroSeminar](https://microseminar.wordpress.com/) video here on [YouTube](https://www.youtube.com/watch?v=aJp6-PTpKvA). We are focused on how bacteria form complex communities within biofilms and how individual cells perceive cues to attach or disperse. We have learned that adaptation involves a sequence of mutations in predictable targets that leads to diversification into ecologically differentiated subpopulations. Often, these subpopulations interact synergistically, with certain types affecting host response and others enhancing resistance. In addition, we now routinely sequence the genomes of hundreds of bacterial isolates from longitudinal samples to define evolutionary forces affecting the courses of infection and the driver mutations, whose functions we work to experimentally identify. This work is supported by the NIH. Studying how different bacterial strains or species coexist and interact when bound together on a surface allows us to explore the origins of multicellular life. We are proud to be part of a [NASA Astrobiology Institute](https://astrobiology.nasa.gov/nai/teams/can-7/umt/) that uses experimental evolution to pursue the goal. 

Interested? Check out the following publications:

1. [Poltak, S., Cooper, V.S. Ecological Succession in Long-Term Experimentally Evolved Biofilms Produces Synergistic Communities. International Society for Microbial Ecology, 2010](https://www.nature.com/ismej/journal/v5/n3/pdf/ismej2010136a.pdf)
2. [Traverse, C.C., Mayo-smith, L.M., Poltak, S. Cooper, V.S. Tangled Bank of Experimentally evolved Burkholderia Biofils Reflects Selection During Chronic Infections. PNAS, 2013](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3549113/)
3. [Flynn, K., Dowell, G., Johnson, T.M., Koestler, B.J., Waters, C.M., Cooper, V.S. Evolution of Ecological Diversity in Biofilms of Pseudomonas aeruginosa by Altered Cyclic Diguanylate Signaling. Journal of Bacteriology, 2016](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5019052/pdf/zjb2608.pdf)
4. [Ellis, C.N., Traverse, C.C., Buskirk, S.W., Cooper, V.S. Character Displacement and the Evolution of Niche Complementarity in a Model Biofilm Community. Evolution, 2014](http://onlinelibrary.wiley.com/doi/10.1111/evo.12581/epdf)

<br>

***

<br>

<a name="tradeoff"></a>

# 3. Ecological Tradeoffs

<div class="research_fig">
    <img src="/images/cluster.png" />
    <p>Fitness Landscapes in a selection environment <br>
    <a href="http://journals.plos.org/plosgenetics/article/file?id=10.1371/journal.pgen.1003426&type=printable">The Environment Affects Epistatic Interactions to Alter the Topology of an Empirical Fitness Landscape. K.M. Flynn. et al. PLOS Genetics, 2013</a></p>
</div>

Two major population-genetic forces can produce ecological specialization throughout life: antagonistic pleiotropy (genetic tradeoffs favored by selection) and mutation accumulation (drift eroding unused traits). We study how these forces interact by studying how mutations interact with the external environment and with other mutations (epistasis). For instance, we work to quantify the distribution of effects of beneficial mutations (DFE) in multiple environments ii) fitness effects of spontaneous mutations nearly unseen by natural selection and iii) fitness of mutations favored in the structured and fluctuating environments of bacterial biofilms. This work is supported by NIH and NASA.

Interested? Check out the following publications:

1. [Dillon, M.M., Rouilard, N.P., Dam, B., Gallet, R., Cooper, V.S. Diverse phenotypic and genetic responses to short‚Äêterm selection in evolving Escherichia coli populations. Evolution, 2016](https://www.biorxiv.org/content/biorxiv/early/2015/09/17/027086.full.pdf)
2. [Dillon, M.M., Cooper, V.S. The Fitness Effects of Spontaneous Mutations Nearly Unseen by Selection in a Bacterium with Multiple Chromosomes. Genetics, 2016](http://www.genetics.org/content/204/3/1225)
3. [Wang, Y., et al. Benefit of Transferred Mutations is better Predicted by the Fitness of Recipients than by their Ecological or Genetic Relatedness. PNAS, 2016](http://www.pnas.org/content/113/18/5047.full.pdf)

<br>

***

<br>

<a name="genome_evo"></a>

# 4. Why genome regions evolve at different rates

<div class="research_fig">
    <img src="/images/mut_rates.png" />
    <p>Overall base-substitution and indel mutation rates from the origion of replication oriC in B. cenocepacia. <br>
    <a href="http://www.genetics.org/content/genetics/200/3/935.full.pdf">The rate and molecular spectrum of spontaneous mutations in the GC-Rich multichromosome genome of Burkholderia cenocepacia. M.M. Dillon, et al. Genetics, 2015</a></p>
</div>

In bacterial genomes with multiple chromosomes, smaller, secondary chromosomes evolve more rapidly. Intra-genome variation in the rates and spectra of mutations could provide a mechanism by which selection acts on genome organization to influence both the origin of genetic variation as well as its fate. We use mutation-accumulation experiments (MA) paired with whole-genome-sequencing (WGS) to capture the mutational process of various bacterial species with unprecedented resolution. We are exploring the root causes of this variation as it may bear on variation in genome stability in all organisms. 

Interested? Check out the following publications:

1. [Cooper, V.S., S. Vohr, S. Wrocklage, and P.J. Hatcher. Why Genes Evolve Faster on Secondary Chromosomes in Bacteria. PLoS Computational Biology, 2010](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2848543/pdf/pcbi.1000732.pdf)
2. [Flynn, K.M., S.H. Vohr, P.J. Hatcher, Cooper, V.S. Evolutionary Rates and Gene Dispensability Associate with Replication Timing in the Archaeon Sulfolobus islandicus. Genome Biology and Evolution, 2010](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3000693/pdf/evq068.pdf)
3. [Dillon, M.M., Sung, W., Lynch, M., Cooper, V.S. The Rate and Molecular Spectrum of Spontaneous Mutations in the GC-Rich Multichromosome Genome of Burkholderia cenocepacia. Genetics, 2015](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4512553/pdf/935.pdf)
4. [Dillon, M.M., Sung, W., Sebra, R., Lynch, M., Cooper, V.S. Genome-Wide Biases in the Rate and Molecular Spectrum of Spontaneous Mutations in Vibrio cholerae and Vibrio fischeri. MBE 2016](https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/msw224)

<br>

***

<br>

<a name="evolvingstem"></a>

# 5. EvolvingSTEM

<div class="research_fig">
    <img src="/images/stem_learning.png" />
    <p>EvolvingSTEM</p>
</div>

We have developed and direct a program called [EvolvingSTEM](http://www.evolvingstem.org/) that enables hundreds of high school students to learn about evolution and genetics by hands-on research. In one week, students conduct an evolution experiment with the benign biofilm-forming bacterium Pseudomonas fluorescens, which begins as a clone and ends as an obviously diverse and adapted population. This curriculum increases learning about evolution, ecology, and heredity statistically better than standard non-laboratory-based curricula, it is targeted towards Next-Generation Science Standards, and even increases student motivation to pursue STEM-based careers. It was featured in 5 schools in New Hampshire, two of which continue this curriculum, and is now ongoingat two Pittsburgh-area schools. It is currently supported by a seed award from the Sprout Fund and we seek broader support to refine and distribute this program nationwide.
