---
title: "Research"
permalink: /research/areas/
author_profile: false
layout: research
collection: research
toc: true
toc_label: "Research Areas"
toc_icon: "flask"
toc_sticky: true
excerpt: "Translating inferences from evolutionary biology to improve human health and enable learning through authentic experimentation."
header:
    overlay_image: /images/notebook.png
    overlay_filter: rgba(36, 42, 40, 0.6)
---

<h1>As a lab we are interested in the following areas:</h1>

1. [Mechanisms of beneficial mutations](#mechanism)
2. [Evolution during Infections](#infection_evo)
3. [Ecological-evolutionary dynamics](#eco_evo)
4. [Adaptive dynamics within host-associated microbiomes](#microbiome)
5. [EvolvingSTEM](#evolvingstem)

<br>

<a name="mechanism"></a>
# 1. Mechanisms of beneficial mutations

For years, researchers like us have used microbes as model systems to understand how evolutionary processes work. Increasingly, we can also use evolution to understand how microbes work, as a potent genetic screen for mutations that increase fitness in any imaginable environment. These mutations affect genes and systems that often yield the best adaptations and usually teach us new, surprising, fundamental attributes of microbial biology. For example, [we discovered](https://journals.asm.org/doi/10.1128/mbio.00287-23) that nonsynonymous mutations in the essential tRNA-modifying gene tilS that nearly eliminate its primary function somehow increase fitness in certain environments, pointing toward a new secondary function. We’ve also identified [novel mechanisms of antimicrobial resistance](https://journals.asm.org/doi/10.1128/mBio.00932-20https://journals.asm.org/doi/10.1128/mBio.00932-20) and [new regulatory functions of enzymes involved in cyclic-di-GMP turnover](https://www.pnas.org/doi/10.1073/pnas.2008540117) that enable functional diversity in biofilms. More recently we’ve collaborated with colleagues who have developed animal models for experimental evolution, which has led us to discover how [certain beneficial mutations produce adaptations](https://journals.asm.org/doi/full/10.1128/mSystems.00352-20) to the host environment. We continue to investigate why mutations in certain genes are repeatedly selected in evolution experiments or chronic infections and how they work. 

This work is supported by grants from NSF and NIH.

<div class="research_fig">
    <img src="/images/mhatre_fig7.jpg" />
    <p>Possible ways <i>rpfR</i> mutants can influence biofilm phenotypes. 
    <a href="https://doi.org/10.1073/pnas.2008540117">Figure from Mhatre et al. PNAS, 2020</a></p>
</div>

<br>

***

<br>

<a name="infection_evo"></a>

# 2. Evolution during Infections

Microbes often adaptively evolve when establishing infections by escaping components of the immune system and subsequently, antimicrobial treatment. We are interested in how bacterial populations evolve to escape antibiotic and host pressures during infections, with the goal of improving prediction of antimicrobial resistance (AMR). We study the evolution of drug resistance and pathogenicity [in near real time through <i>in vitro</i>](https://elifesciences.org/articles/47612) and [<i>in vivo</i> experimental evolution](https://journals.asm.org/doi/full/10.1128/mSystems.00352-20)  using [comparative genomics](https://journals.asm.org/doi/full/10.1128/mSystems.00029-16) and analyses of longitudinal deep-sequenced populations. Our research questions include: (1) [mechanisms of drug resistance](https://journals.asm.org/doi/full/10.1128/mBio.00932-20) and [treatment failure](https://www.nature.com/articles/s41564-022-01126-8), (2) the impact of the [biofilm lifestyle on resistance](https://elifesciences.org/articles/47612), (3) how host immune state influences resistance evolution, and (4) comparative genomics of multi-drug resistant clinical isolates. 

This work is supported by the NIH U19AI158076 CARBIRU project and by the PA Department of Health.

<div class="research_fig">
    <img src="/images/infection_evo_fig.jpeg" />
    <p><i>Pseudomonas aeruginosa</i> from sinuses of people with cystic fibrosis show diverse pathoadaptive traits. Figure from 
    <a href="https://doi.org/10.1016/j.celrep.2021.109829">Armbruster et al (2021) <i>Cell Reports</i></a>.</p>
</div>

<br>

***

<br>

<a name="eco_evo"></a>

# 3. Ecological-evolutionary dynamics

Microbes exist within complex communities where both abiotic factors and inter-/intra-species interactions influence evolutionary trajectories. These [environmental factors impose selective forces on communities](http://doi.wiley.com/10.1002/evl3.75). Likewise, individuals in a community can modify their environment and indirectly influence selective pressures applied to the community. We are interested in understanding such ecological and evolutionary dynamics at a community scale, especially in the context of [biofilms](http://dx.doi.org/10.1038/ismej.2010.136), polymicrobial communities, and [active bacteriophages](https://www.science.org/doi/full/10.1126/sciadv.abh1489).

<div class="research_fig">
    <img src="/images/eco_evo_fig.jpeg" />
    <p>Muller plots showing the changes in <i>Pseudomonas aeruginosa</i> genotype frequency over time. Muller plots were created in <a href="https://github.com/cdeitrick/Lolipop">Lolipop</a>, a bioinformatic tool developed in our lab. Figure from 
    <a href="https://doi.org/10.1093/molbev/msab248">Harris, Flynn, & Cooper (2021) <i>Mol Biol Evol</i></a>.</p>
</div>

<br>

***

<br>

<a name="microbiome"></a>

# 4. Adaptive dynamics within host-associated microbiomes

Like any disrupted ecosystem, when a host-associated microbiome is far from equilibrium, certain community members may be selected to inhabit vacant niches in which they are relatively maladapted. These lineages may undergo rapid adaptive evolution that can be tracked by longitudinal genome sequencing, alongside more typical unbiased measures of the community as it remodels. We study these processes in a wide range of systems with experts focused on particular diseases or symbioses, including:
- in the upper and lower airways of [persons with cystic fibrosis](https://www.sciencedirect.com/science/article/pii/S2211124721012936), 
- during establishment of [chronic wounds](https://journals.asm.org/doi/full/10.1128/mBio.01698-19),
- during chronic infections of [prosthetic joint replacement](https://onlinelibrary.wiley.com/doi/full/10.1002/jor.25031)
- during colonization of the infant gut
- during [symbiosis with the Hawaiian bobtail squid](https://elifesciences.org/articles/24414)
and even among the virome within bacterial cells

(NIAID F31AI179118, NHLBI R33H137077 w/ Bomberger, R01DK120697 w/ Hand, R01AI165519 w/ Van Tyne, R01HL162964 w/ Zemke)


<!-- <div class="research_fig">
    <img src="/images/placeholder-image.png" />
    <p>Placeholder caption</p>
</div> -->

<br>

***

<br>

<a name="evolvingstem"></a>

# 5. EvolvingSTEM

We created EvolvingSTEM, a research-education partnership program that uses evolution-in-action to capture the imagination of middle and high school biology students. We partner with classrooms across the US to engage students in an authentic laboratory experience. They evolve populations of Pseudomonas fluorescens under selection for biofilm growth and observe conspicuous changes to colony morphology in as little as one week. Our goals are to (1) [increase understanding of key life science topics, including evolution, microbiology, and genetics,](https://evolution-outreach.biomedcentral.com/articles/10.1186/s12052-019-0103-4) (2) provide training in essential biotechnology skills, and (3) inspire an enduring interest in science. Students gain confidence in STEM topics through hands-on experiences and drawing connections from their bench-top experiments to real-life medical applications. In addition, we train teachers to effectively implement our program in their classrooms through an immersive 8-week summer research experience hosted in our laboratory through support from the National Science Foundation (NSF BIORETS 2147075, NIH SEPA R25AI180989).

<div class="research_fig">
    <img src="/images/EvolvingSTEM_class.png" alt="High school students doing hands-on evolution experiments in class through the EvolvingSTEM program" />
    <p>High school students doing hands-on evolution experiments in class through the EvolvingSTEM program.</p>
</div>

Interested? Learn more at the [EvolvingSTEM's official website](https://evolvingstem.org/)