<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Cooper Lab - Research</title>

    <!-- Bootstrap Core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom CSS -->
    <link href="css/landing-page.css" rel="stylesheet">

    <!-- Custom Fonts -->
    <link href="font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700,300italic,400italic,700italic" rel="stylesheet" type="text/css">

<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Cooper Lab - Research</title>

    <!-- Bootstrap Core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom CSS -->
    <link href="css/landing-page.css" rel="stylesheet">

    <!-- Custom Fonts -->
    <link href="font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700,300italic,400italic,700italic" rel="stylesheet" type="text/css">

    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->

</head>

<body>

    <!-- Navigation -->
    <nav class="navbar navbar-default navbar-fixed-top topnav" role="navigation">
        <div class="container topnav">
            <!-- Brand and toggle get grouped for better mobile display -->
            <div class="navbar-header">
                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                    <span class="sr-only">Toggle navigation</span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                </button>
                <a class="navbar-brand topnav" href=index.html>Home</a>
            </div>
            <!-- Collect the nav links, forms, and other content for toggling -->
            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                <ul class="nav navbar-nav navbar-right">
                    <li>
                        <a href="research.html">About</a>
                    </li>
                    <li>
                        <a href="sequencing.html">Services</a>
                    </li>
                    <li>
                        <a href="contact.html">Contact</a>
                    </li>
                </ul>
            </div>
            <!-- /.navbar-collapse -->
        </div>
        <!-- /.container -->
    </nav>

    <!-- Header -->
    <a name="about"></a>
    <div class="research-header"> 
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                        <h1>Research</h1>
        	</div>
            </div>

        </div>
        <!-- /.container -->

    </div>
    <!-- /.intro-header -->
	<div class="content-section-b">

        <div class="container">
            <div class="row">
		    <hr class="spacing-research">
                    <div class="clearfix"></div>
		    <center><h9>As a lab we seek to answer the following five questions:</h9></center>
	            <br><center><h10><ul type="1" class="ol">
                    <li><a href = "#resistance" class="nounderline">How do microbes adaptively evolve when colonizing eukaryotic hosts, either as pathogens or symbionts? Can we predict these dynamics and identify driver mechanisms towards resistance or pathogenicity to guide therapy?</a></li><br>
                    <li><a href = "#communities" class="nounderline">How do bacteria evolve and form communities within biofilms?  What does this teach us about nascent multicellularity</a></li><br>
                    <li><a href = "#ecological" class="nounderline">How and why do ecological tradeoffs evolve?</a></li><br>
                    <li><a href = "#rates" class="nounderline">Why do genome regions replicated at different times evolve at different rates?</a></li><br>
    		    <li><a href = "#stem" class="nounderline">Evolution is best taught by hands-on experimentation.  How do we deliver this on a massive scale?</a></li>
		    </ul></h10></center><br>
		    <hr class="spacing-research" id="resistance"><br>
		    <div class="col-lg-8">
                    <h2 class="section-heading">Host adaptation and the evolution of antimicrobial resistance</h2>
                    <p class="lead">Our laboratory is developing the translational utility of evolutionary inference 
                    		    in medicine using new experimental, genomic and bioinformatic methods. Our focus is the 
                    		    evolution-in-action that occurs during human infections, particularly in the airway of 
                    		    Cystic Fibrosis patients, and we also use laboratory models of this process. 
                    		    We run a <a href="sequencing.html">sequencing facility</a> with an Illumina NextSeq and Oxford Nanopore 
                    		    and collaborate with many investigators to identify mutations under positive selection during 
                    		    infections and their functional significance, with a focus on adaptations to antibiotics 
                    		    and to biofilm formation. We also study adaptation by symbionts to animal hosts such as the 
                    	            Hawaiian bobtail squid, nematodes, and the medicinal leech.  This work is supported by NIH, 
                    		    NSF, and by Gilead.  Interested?  Check out our following publications:</p>
		<ul type=1>
		<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5466423/pdf/elife-24414.pdf">Pankey, M.S., Foxall, R.L., Ster, I.M., Perry, L.A., Schuster, B.M., Donner, R., Coyle, M., Cooper, V.S., Whistler, C.A. Host-selected Mutations Converging on a Global Regulator Drive on Adaptive Leap Towards Symbiosis in Bacteria. eLife, 2017</a></li>
                <li><a href="https://academic.oup.com/cid/article-abstract/doi/10.1093/cid/cix182/3052769/Ceftolozane-tazobactam-for-the-treatment-of?redirectedFrom=fulltext">Haidar, G., Phillips, N.J., Shields, R.K., Snyder, D., Cheng, S., Potoski, B.A., Doi, Y., Hao, B., Press, E.G., Cooper, V.S., Clancy, C.J., Nguyen M.H. Ceftolozane-Tazobactam for the Treatment of Multidrug-Resistant Pseudomonas aeruginosa Infections: Clinical Effectiveness and Evolution of Resistance, Clinical Infectious Diseases, 2017</a></li>
                <li><a href="http://mbio.asm.org/content/8/1/e02124-16.full">Honsa, E.S., Cooper, V.S., Mhaissen, M.N., Frank, M., Shaker, J., Iverson, A., Rubnitz, J., Hayden, R.T., Lee, R.E., Rock, C.O., Tuomanen, E.I., Wolf, J., Rosch, J.W. RelA Mutant Enterococcus faecium with Multiantibiotic Tolerance Arising in an Immunocompromised Host. mBio, 2017</a></li>
                <li><a href="http://msystems.asm.org/content/msys/1/3/e00029-16.full.pdf>Silva, I.N., Santos, P.M., Santos, M.R., Zlonsnik J., Speert, D.P., Buskirk, S.W., Bruger, E.L., Waters, C.M., Cooper, V.S., Moreira, L.M. Long-Term Evolution of Burkholderia multivorans during a Chronic Cystic Fibrosis Infection Reveals Shifting Forces of Selection. mSystems, 2016</a></li>
		</ul>
                </div>
                <div class="col-lg-4">
                    <img class="img-responsive" src="holder" alt="">
                </div>
            </div>

        </div>
        <!-- /.container -->

    </div>
    <!-- /.content-section-b -->   
   
    <div class="content-section-c">

        <div class="container" id="communities">

            <div class="row">
                <div class="col-lg-8">
                    <hr class="section-heading-spacer">
                    <div class="clearfix"></div>
                    <h2 class="section-heading">Biofilms and the origins of multicellularity</h2>
                    <p class="lead">Check out our <a href="https://microseminar.wordpress.com/">MicroSeminar</a> video here on 
                    	  	    <a href="https://www.youtube.com/watch?v=aJp6-PTpKvA">YouTube</a> We are focused on how 
                    		    bacteria form complex communities within biofilms and how individual cells perceive cues to 
                    	 	    attach or disperse. We have learned that adaptation involves a sequence of mutations in 
                    		    predictable targets that leads to diversification into ecologically differentiated 
                    		    subpopulations. Often, these subpopulations interact synergistically, with certain types 
                    		    affecting host response and others enhancing resistance. In addition, we now routinely 
                    		    sequence the genomes of hundreds of bacterial isolates from longitudinal samples to define 
                    		    evolutionary forces affecting the courses of infection and the driver mutations, whose 
                    		    functions we work to experimentally identify. This work is supported by the NIH.
				    Studying how different bacterial strains or species coexist and interact when bound together 
				    on a surface allows us to explore the origins of multicellular life. We are proud to be part 
				    of a <a href="https://astrobiology.nasa.gov/nai/teams/can-7/umt/">NASA Astrobiology Institute</a> 
				    that uses experimental evolution to pursue the goal.  Interested?  Check out our following publications:</p>
                <ul type=1>
		<li><a href="https://www.nature.com/ismej/journal/v5/n3/pdf/ismej2010136a.pdf">Poltak, S., Cooper, V.S. Ecological Succession in Long-Term Experimentally Evolved Biofilms Produces Synergistic Communities. International Society for Microbial Ecology, 2010</a></li>
                <li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3549113/">Traverse, C.C., Mayo-smith, L.M., Poltak, S. Cooper, V.S. Tangled Bank of Experimentally evolved Burkholderia Biofils Reflects Selection During Chronic Infections. PNAS, 2013</a></li>
                <li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5019052/pdf/zjb2608.pdf">Flynn, K., Dowell, G., Johnson, T.M., Koestler, B.J., Waters, C.M., Cooper, V.S. Evolution of Ecological Diversity in Biofilms of Pseudomonas aeruginosa by Altered Cyclic Diguanylate Signaling. Journal of Bacteriology, 2016</a></li>
                <li><a href="http://onlinelibrary.wiley.com/doi/10.1111/evo.12581/epdf">Ellis, C.N., Traverse, C.C., Buskirk, S.W., Cooper, V.S. Character Displacement and the Evolution of Niche Complementarity in a Model Biofilm Community. Evolution, 2014</a></li>
		</ul>
		</div>
                <div class="col-lg-4">
                    <img class="img-responsive" src="holder" alt="">
                </div>
            </div>

        </div>
        <!-- /.container -->

    </div>
    <!-- /.content-section-c -->



    <div class="content-section-b">

        <div class="container" id="ecological">

            <div class="row">
                <div class="col-lg-8">
                    <hr class="section-heading-spacer">
                    <div class="clearfix"></div>
                    <h2 class="section-heading">Ecological Tradeoffs</h2>
                    <p class="lead">Two major population-genetic forces can produce ecological specialization throughout life: antagonistic 
				    pleiotropy (genetic tradeoffs favored by selection) and mutation accumulation (drift eroding unused traits). We 
				    study how these forces interact by studying how mutations interact with the external environment and with other 
				    mutations (epistasis). For instance, we work to quantify the distribution of effects of beneficial mutations (DFE) 
				    in multiple environments ii) fitness effects of spontaneous mutations nearly unseen by natural selection and iii) 
				    fitness of mutations favored in the structured and fluctuating environments of bacterial biofilms. This work is 
				    supported by NIH and NASA.  Interested?  Check out our following publications:</p>
		<ul type="1">
		<li><a href="http://www.biorxiv.org/content/biorxiv/early/2015/09/17/027086.full.pdf">Dillon, M.M., Rouilard, N.P., Dam, B., Gallet, R., Cooper, V.S. Diverse phenotypic and genetic responses to short‐term selection in evolving Escherichia coli populations. Evolution, 2016</a></li>
		<li><a href="http://www.genetics.org/content/204/3/1225">Dillon, M.M., Cooper, V.S. The Fitness Effects of Spontaneous Mutations Nearly Unseen by Selection in a Bacterium with Multiple Chromosomes. Genetics, 2016</a></li>
		<li><a href="http://www.pnas.org/content/113/18/5047.full.pdf"> Wang, Y., Arenas, C.D., Stoebel, D.M., Flynn, K., Knapp, E., Dillon, M.M., Wünsche, A., Hatcher, P.J., Moore, F., Cooper, V.S., Cooper T.F. Benefit of Transferred Mutations is better Predicted by the Fitness of Recipients than by their Ecological or Genetic Relatedness. PNAS, 2016</a></ol>
		</ol>
                </div>
                <div class="col-lg-4">
                    <img class="img-responsive" src="holder" alt="">
                </div>
            </div>

        </div>
        <!-- /.container -->

    </div>
    <!-- /.content-section-b -->



    <div class="content-section-c">

        <div class="container" id="rates">

            <div class="row">
                <div class="col-lg-8">
                    <hr class="section-heading-spacer">
                    <div class="clearfix"></div>
                    <h2 class="section-heading">Why genome regions evolve at different rates</h2>
                    <p class="lead">In bacterial genomes with multiple chromosomes, smaller, secondary chromosomes evolve more rapidly. Intra-genome 
				    variation in the rates and spectra of mutations could provide a mechanism by which selection acts on genome 
				    organization to influence both the origin of genetic variation as well as its fate.  We use mutation-accumulation 
				    experiments (MA) paired with whole-genome-sequencing (WGS) to capture the mutational process of various bacterial 
				    species with unprecedented resolution. We are exploring the root causes of this variation as it may bear on variation
				    in genome stability in all organisms.  Interested?  Check out our following publications:</p>
		<ul type="1">
		<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2848543/pdf/pcbi.1000732.pdf">Cooper, V.S., S. Vohr, S. Wrocklage, and P.J. Hatcher. Why Genes Evolve Faster on Secondary Chromosomes in Bacteria. PLoS Computational Biology, 2010</a></li>
		<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3000693/pdf/evq068.pdf">Flynn, K.M., S.H. Vohr, P.J. Hatcher, Cooper, V.S. Evolutionary Rates and Gene Dispensability Associate with Replication Timing in the Archaeon Sulfolobus islandicus. Genome Biology and Evolution, 2010</a></li>
		<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4512553/pdf/935.pdf">Dillon, M.M., Sung, W., Lynch, M., Cooper, V.S. The Rate and Molecular Spectrum of Spontaneous Mutations in the GC-Rich Multichromosome Genome of Burkholderia cenocepacia. Genetics. 2015;200(3):935-46</a></li>
		<li><a href="https://watermark.silverchair.com/api/watermark?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAAekwggHlBgkqhkiG9w0BBwagggHWMIIB0gIBADCCAcsGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM8aafzMadYSej6eStAgEQgIIBnHY1WhUXjZX6mgqwBGRrxxj9i0XaPdQe4v_EjNTRZLK0Jsb2M8f3241xsiRK1tCmLqxssaQhuRjJ_slUGZ7WHM576Jfm5LRdSSLrrwhWOCPpuSXP5AJsXLKAHDbE_heFB0SI3LWqj6lhP5zRvXZs77ZWqpGTut0EF5suaHkUVfDTIRLnhdXY531SO2KX1gsJnjKoNWSXQ2g0_-3RPIApnAEpe_ykNzgus_SnBGBC42--yoQi4mKdbKYjJUI3fr7LZ1VlHw4UCzo6-w7Wd6Sa_tyW5jYJ3oqIGAXbMXs9pHQSo206RFNpKAdpFri0XAZk0yPD0Fd2crwqCyH_hIM-pt635HXEQkbCOyoK7jkY3DyREV5VxrAwLYItcEKtLoiQFiewyOp15ikXyqy49agkBt32krPkoiuA8tux-zARdPkzDZlz-7uONdWMv-Gld3ZYwTEP-3_Ukmbh-dkE69R9u9ODUb5rt3FZlGnyR42G5yCmoclhrh8ujJ4Yy3apDG-qk_Kq_wUXUg8u7IzPfGZzrrF7XwGOm4B2-Gru7mg">Dillon, M.M., Sung, W., Sebra, R., Lynch, M., Cooper, V.S. Genome-Wide Biases in the Rate and Molecular Spectrum of Spontaneous Mutations in Vibrio cholerae and Vibrio fischeri 2016. MBE 2016</a></li>
		</ul>
		</div>
                <div class="col-lg-4">
                   <br><br><br> <img class="img-responsive" src="img/mut_rates.png" alt="">
                </div>
            </div>

        </div>
        <!-- /.container -->

    </div>
    <!-- /.content-section-c -->




    <div class="content-section-b">

        <div class="container" id="stem">

            <div class="row">
                <div class="col-lg-8">
                    <hr class="section-heading-spacer">
                    <div class="clearfix"></div>
                    <h2 class="section-heading">EvolvingSTEM</h2>
                    <p class="lead">We have developed and direct a program called EvolvingSTEM that enables hundreds of high school students to learn 
		     		    about evolution and genetics by hands-on research. In one week, students conduct an evolution experiment with the 
				    benign biofilm-forming bacterium *Pseudomonas fluorescens*, which begins as a clone and ends as an obviously diverse 
				    and adapted population. This curriculum increases learning about evolution, ecology, and heredity statistically 
				    better than standard non-laboratory-based curricula, it is targeted towards Next-Generation Science Standards, and 
				    even increases student motivation to pursue STEM-based careers. It was featured in 5 schools in New Hampshire, two 
				    of which continue this curriculum, and is now ongoingat two Pittsburgh-area schools. It is currently supported by a 
				    seed award from the Sprout Fund and we seek broader support to refine and distribute this program nationwide.</p>


                </div>
                <div class="col-lg-4">
                    <img class="img-responsive" src="img/stem_learning.png" alt="">
                </div>
            </div>

        </div>
        <!-- /.container -->

    </div>
    <!-- /.content-section-b -->

    <!-- Footer -->
    <footer>
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                    <ul class="list-inline">
                        <li>
                            <a href="index.html">Home</a>
                        </li>
                        <li class="footer-menu-divider">&sdot;</li>
                        <li>
                            <a href="research.html">About</a>
                        </li>
                        <li class="footer-menu-divider">&sdot;</li>
                        <li>
                            <a href="sequencing.html">Services</a>
                        </li>
                        <li class="footer-menu-divider">&sdot;</li>
                        <li>
                            <a href="contact.html">Contact</a>
                        </li>
                    </ul>
                    <p class="copyright text-muted small">Copyright &copy; The Cooper Lab, University of Pittsburgh, 2017. All Rights Reserved</p>
                </div>
            </div>
        </div>
    </footer>

</body>
</html>
